tests/causal_estimators/test_causalml_estimator.py

```diff
@@ -36,6 +30,18 @@ def test_LRSRegressor(self):
             num_samples=1000,
             treatment_is_binary=True
         )
+    

```

Comment:
Thanks for adding the fixture. But for this to work, I think you also need to add it as a fixture to the test-class.

Reply:
cool